systemservices/ethwallet/ethwallet/sign.go

```diff
@@ -0,0 +1,63 @@
+package ethwallet
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/mesg-foundation/core/client/service"
+	"github.com/mesg-foundation/core/systemservices/ethwallet/x/xgo-ethereum/xaccounts"
+)
+
+type signInputs struct {
+	Address     string       `json:"address"`
+	Passphrase  string       `json:"passphrase"`
+	Transaction *transaction `json:"transaction"`
+	ChainID     int64        `json:"chainID"`
```

Comment:
ChainID is *big.Int 
https://github.com/ethereum/go-ethereum/blob/ecb781297bfc891f4ff26bdf3fda362744bbb3e3/params/config.go#L148

Somewhere I saw uint64, but not int64